packages/service-clients/tinylicious-client/api-report/tinylicious-client.alpha.api.md

```diff
@@ -6,10 +6,10 @@
 
 export { CompatibilityMode }
 
-// @beta
+// @public
 export type ITinyliciousAudience = IServiceAudience<TinyliciousMember>;
 
-// @beta @sealed
+// @public @sealed
```

Comment:
FWIW, I think almost all of these interfaces could/should be `@sealed` - might be worth considering?

Reply:
I'll do a quick follow-up PR.

Reply:
Actually, for interfaces like `User` and `Member`, I think we're probably okay with extension (correct me if I'm wrong). But the others probably make sense.